### ISSUE TITLE
Radarr urlbase and bindaddress not getting set

### ISSUE THREAD

**Issue by KarmaPoliceT2** [QuickBox/QB#15]
I can see the sed commands in the install package, but for some reason they had no effect on install from the dashboard... need to investigate why not... 

These not being set properly caused the "Radarr ver." screen to appear when clicking the link from the dashboard.

**Comment by JMSDOnline** [comment]
The `config.xml` is not being created per the `/home/${username}/.config/Radarr/` directory. This will fail especially when the directory doesn't exist. I have an install playing on a v2.5.0 install where it was working. I'll look to see what changes happened since. This one is a puzzle as @PastaGringo created this installer package.

**Comment by JMSDOnline** [comment]
Addressed a few issues and pushed to QB[master] see commit here: c568ddb70c7b2c74b1b5fab2251f48726fd8147e

May want to run a few tests, I have had success on 3 separate runs now, both from fresh install and current installs.

**Comment by KarmaPoliceT2** [comment]
Think there is still an issue:

Line 76 is currently:
`+chown -R quickbox:quickbox /home/${username}/.config`
I think it should be: 
`+chown -R ${username}:${username} /home/${username}/.config`

**Comment by userdocs** [comment]
I was looking at the install script and it seems that you load radarr to create the config then edit it. I found this method very inconsistent when creating couch/sick/sonarr/radarr installations. 

1: With no configs generated if the server is not shutdown properly and the wait time for this proper shutdown is not consistent or predictable.

2: Or configs being overwritten tables in the database at load and shutdown for various reasons, such as config being edited before the app issues the save to config.

What I did to get a consistent outcome was to use a template config that inserts the requires database tables at first run and guarantees it will save the desired config on exit.

Radarr Config.

https://github.com/userdocs/userdocs/blob/master/Remote_Programs/Radarr/configs/config.xml

1: Download the template
2: edit the template
3: Load Radarr.
4: Profit. Consistent outcome every time.

The sonarr config is almost identical.

https://github.com/userdocs/userdocs/blob/master/Remote_Programs/Sonarr/configs/config.xml

Jakett works the same.

https://github.com/userdocs/userdocs/blob/master/Remote_Programs/Jackett/configs/ServerConfig.json

This works with Couchpotatoe and Sickrage as well.

This is from a mono apps script I wrote for whatbox/feral slots after various scripts for couch/sick et al.

https://github.com/userdocs/userdocs/blob/master/Remote_Programs/Mono/scripts/install.monoapps.sh

I get the exact outcome I want every time.
